### ISSUE TITLE
Byline is missing for anonymous user

### ISSUE THREAD

**Issue by hvelarde** [plone/Products.CMFPlone#1556]
Plone 5.0.4 and again no document byline visible.

according to 1d752d9b83c72e6a7a0aacf07ec85e4a5728004a the document byline viewlet was hidden without any kind of notice.

enabling it using the `@@manage-viewlets` makes it visible to authenticated users but not to anonymous users.

Refs. #668 and #741

**Comment by hvelarde** [comment]
I forgot to enable the display of this viewlet on **Site Setup** -> **Security Settings**:

![selection_068](https://cloud.githubusercontent.com/assets/341393/15046074/703e53e0-12b4-11e6-85ab-170358e3d4a7.png)

in fact this could solve part of the issue if we display the viewlet just for anonymous users.
